cfpb/cfgov-refresh
# Fix test paths for code coverage

Istanbul code coverage wasn't including /apps/ path in JS unit tests.

## Changes

- Moves unit tests to /js/ directory so that both /js/ and /apps/**/js directories are processed by Istanbul.
- Normalizes name and location of BASE_JS_PATH variable.

## Testing

1. Run `gulp test:unit`
2. Open `unit_test_coverage/lcov-report/index.html` and note that apps/ or js/ prefix spec paths.
